src/library.js

```diff
@@ -3446,6 +3446,14 @@ mergeInto(LibraryManager.library, {
 #endif
   },
 
+  emscripten_runtime_keepalive_push: '$runtimeKeepalivePush',
+  emscripten_runtime_keepalive_pop: '$runtimeKeepalivePop',
+  emscripten_runtime_keepalive_check: function() {
+    // keepRuntimeAlive is a runtime function rather than a library function,
```

Comment:
What is the difference between a runtime function and a library function?

Reply:
A runtime function is just a few function in the JS preamble/postamble/etc.  There is no way to include or exclude runtime functions.

Library functions are the ones defined in these specifically-formed library files (mostly `src/library.js` and `src/library_*.js`.   Library functions are included on demand and never included unless somebody depends on them.  We tend to prefer library functions these day because they have no impact on users codesize unless they are actually needed.